### PR TITLE
Css selectors

### PR DESCRIPTION
**Pull request by Kadif-Ravik**
### Description
first PR. Description rewrite, grammar and punctuation. 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/selectors/selectors.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ p {
 
 ## Selecting by Attribute
 
-Most commonly, the `class` and `id` attributes are assigned to elements for styling purposes. An `id` cannot start with a number.
+The `class` and `id` attributes are generally assigned for styling different elements on the same page and linking to sections or other pages of the site.
```
Comment (SSwiniarski) [inline content/css/concepts/selectors/selectors.md:53]:
```suggestion
The `class` and `id` attributes are generally assigned for styling different elements. An `id` cannot start with a number.
```
`class` is not used for linking. And the linking use case is outside the use as selectors. Also deleted a relevant sentence.

File: content/css/concepts/selectors/selectors.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ CatalogContent:
 
 **Selectors** are used to style [HTML elements](https://www.codecademy.com/resources/docs/html/elements) according to their type and/or attributes.
 
-All markups can be selected with the `*` selector:
+All markups can be selected with the `*` selector.
```
Comment (SSwiniarski) [inline content/css/concepts/selectors/selectors.md:18]:
```suggestion
All markups can be selected with the `*` selector:
```
We typically introduce example codeblocks with a colon.

File: content/css/concepts/selectors/selectors.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ The `element-type` must be a valid HTML element.
 
 ### Example
 
-In the following example, all `<p>` elements on the page will be center-aligned, with a red text color:
+In the following example all `<p>` elements on the page will be center-aligned with a red text color.
```
Comment (SSwiniarski) [inline content/css/concepts/selectors/selectors.md:42]:
```suggestion
In the following example all `<p>` elements on the page will be center-aligned with a red text color:
```
We typically introduce example codeblocks with a colon.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Kadif-Ravik I did a first review and left a few comments.

File: content/css/concepts/selectors/selectors.md (hunk 3 above)
Comment (CBID2) [inline content/css/concepts/selectors/selectors.md:42]: [Proofreading]
```suggestion
In the following example, all of the `<p>` elements on the page will be centered with a red text color.
```

File: content/css/concepts/selectors/selectors.md (hunk 1 above)
Comment (CBID2) [inline content/css/concepts/selectors/selectors.md:53]: [Proofreading] 
Hi @Kadif-Ravik! :) Consider using this suggestion as an idea to apply SSwiniarski's suggestions. 
```suggestion
The `class` selector selects elements that are assigned under a specific attribute, a class, given to an HTML element, can be used multiple times, and begins with a period, `.`. Similarly, the `id` selector selects an HTML element that has an `id` attribute. In contrast, the `id` selector is **only** used once and begins with the hashtag symbol, `#`.  
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Kadif-Ravik! :) I just gave a suggestion on how to improve your PR. Hope it helps! :)

File: content/css/concepts/selectors/selectors.md (hunk 2 above)
Comment (Kadif-Ravik) [reply content/css/concepts/selectors/selectors.md:18]: Thanks

**Comment by CBID2** [comment]
Hey @Kadif-Ravik! :) Are you still working on this entry?

**Comment by SSwiniarski** [comment]
@Kadif-Ravik In the interest of moving this PR along, I implemented mine and @CBID2's suggestions so this can go on to a second maintainer review.

**Review by SSwiniarski — APPROVED** [review]

**Review by caupolicandiaz — APPROVED** [review]
@Kadif-Ravik @SSwiniarski I made a few language and formatting edits. Looks good to merge

**Comment by SSwiniarski** [comment]
@Kadif-Ravik Congrats on the new edit!

Live link:
- https://www.codecademy.com/resources/docs/css/selectors

**Pull request merged**